app/controllers/graphql_controller.rb

```diff
@@ -12,4 +12,11 @@ def query
     )
     render json: result
   end
+
+  private
+
+  def rbac_allowed?
+    user.authorized_to?(Rbac::INVENTORY_READ) &&
+      user.authorized_to?(Rbac::COMPLIANCE_READ)
```

Comment:
Would be nice to consume multiple arguments in `authorized_to?` using the splat `*` operator.